src/openrct2/ride/coaster/meta/MaurerRollerCoaster.h

```diff
@@ -0,0 +1,86 @@
+    SET_FIELD(Heights, { 28, 24, 9, 11}),
+    SET_FIELD(MaxMass, 18),
+    SET_FIELD(LiftData, { OpenRCT2::Audio::SoundId::LiftClassic, 4, 6 }),
+    SET_FIELD(RatingsMultipliers, { 52, 36, 10 }),
+    SET_FIELD(UpkeepCosts, { 40, 20, 80, 10, 3, 10 }),
+    SET_FIELD(BuildCosts, { 37.50_GBP, 2.50_GBP, 50}),
+    SET_FIELD(DefaultPrices, { 20, 20 }),
+    SET_FIELD(DefaultMusic, MUSIC_OBJECT_ROCK_1),
+    SET_FIELD(PhotoItem, ShopItem::Photo),
+    SET_FIELD(BonusValue, 80),
+    SET_FIELD(ColourPresets, TRACK_COLOUR_PRESETS(
+        { COLOUR_LIGHT_ORANGE, COLOUR_LIGHT_ORANGE, COLOUR_WHITE },
+        { COLOUR_YELLOW, COLOUR_YELLOW, COLOUR_BLACK },
+        { COLOUR_LIGHT_BLUE, COLOUR_LIGHT_BLUE, COLOUR_DARK_GREEN }
+    )),
+    SET_FIELD(ColourPreview, { SPR_RIDE_DESIGN_PREVIEW_MAURER_ROLLER_COASTER_TRACK, SPR_RIDE_DESIGN_PREVIEW_MAURER_ROLLER_COASTER_SUPPORTS }),
+    SET_FIELD(ColourKey, RideColourKey::Ride),
+    SET_FIELD(Name, "trex_rc"),
+
+    SET_FIELD(RatingsData,
+    {
+        RatingsCalculationType::Normal,
+        { RIDE_RATING(2, 55), RIDE_RATING(2, 40), RIDE_RATING(1, 85) },
+        13,
+        -1,
+        false,
+        {
+            { RatingsModifierType::BonusLength,           6000,             764, 0, 0 },
+            { RatingsModifierType::BonusSynchronisation,  0,                RIDE_RATING(0, 40), RIDE_RATING(0, 05), 0 },
+            { RatingsModifierType::BonusTrainLength,      0,                187245, 0, 0 },
+            { RatingsModifierType::BonusMaxSpeed,         0,                44281, 88562, 35424 },
+            { RatingsModifierType::BonusAverageSpeed,     0,                291271, 436906, 0 },
+            { RatingsModifierType::BonusDuration,         150,              26214, 0, 0 },
+            { RatingsModifierType::BonusGForces,          0,                20480, 23831, 49648 },
+            { RatingsModifierType::BonusTurns,            0,                26749, 34767, 45749 },
+            { RatingsModifierType::BonusDrops,            0,                29127, 46811, 49152 },
+            { RatingsModifierType::BonusSheltered,        0,                25700, 30583, 35108 },
+            { RatingsModifierType::BonusReversedTrains,   0,                2, 10, 12 },
+            { RatingsModifierType::BonusProximity,        0,                20130, 0, 0 },
+            { RatingsModifierType::BonusScenery,          0,                9760, 0, 0 },
+            { RatingsModifierType::RequirementDropHeight, 12,               2, 2, 2 },
+            { RatingsModifierType::RequirementMaxSpeed,   0x70000,          2, 2, 2 },
+            { RatingsModifierType::RequirementNegativeGs, FIXED_2DP(0, 50), 2, 2, 2 },
+            { RatingsModifierType::RequirementNumDrops,   2,                2, 2, 2 },
+            { RatingsModifierType::PenaltyLateralGs,      0,                20480, 23831, 49648 },
```

Comment:
These values appear to have been copied unchanged from the mini coaster, which is a good starting point but maybe it should be changed a bit? If it's meant to be a spinning coaster, for example, perhaps the base excitement and nausea values could go up a bit, or the bonus for turns could increase.

I usually look at multiple existing rides that have similarities, here, I'd look at the mini coaster but also the spinning wild mouse, and compare the spinning wild mouse to the regular wild mouse, and perhaps also look at the spiral coaster. I usually try to come up with ride rating values that are consistent with the existing rides without being a direct copy of any of them.  However, you will find that a lot of the values will not need to change as they're the same for almost everything.